js/App.vue

```diff
@@ -235,11 +230,11 @@
                 <h2 class="heading">
                     List of all {{ displayableGambits.length }} gambits sorted by
                     <select v-model="order" class="heading bg-transparent text-left">
-                        <option value="popularity">Their popularity</option>
-                        <option value="rate">Their winning rate</option>
-                        <option value="trophies">The number of trophies</option>
-                        <option value="videos">The number of videos</option>
-                        <option value="name">Their name</option>
+                        <option value="popularity">POPULARITY</option>
```

Comment:
We should not use caps here.

Reply:
I think it is capitalized by the styling? I added the caps just to remind myself. Is it bad?

Reply:
Yes, they are capitalized by the style, so if we change the style the whole phrase will not be capitalized except the words you capitalized explicitly. For coherance we need to let the style do its job (capitalize or not).

Reply:
That makes so much sense!